client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -526,11 +528,11 @@ class CalypsoifyIframe extends Component< Props & ConnectedProps & ProtectedForm
 					<EditorMediaModal
 						disabledDataSources={ getDisabledDataSources( allowedTypes ) }
 						enabledFilters={ getEnabledFilters( allowedTypes ) }
-						galleryViewEnabled={ false }
+						galleryViewEnabled={ isAddingMediaWithClassicBlock }
 						isGutenberg={ ! classicBlockEditorId }
 						onClose={ this.closeMediaModal }
 						onInsertMedia={ this.insertClassicBlockMedia }
-						single={ ! multiple }
+						single={ isAddingMediaWithClassicBlock ? false : ! multiple }
```

Comment:
Nitpick: I love love ternary operators (and kudos for the use of bang-bang), but this line fried my brain a little. I'm not sure where `multiple` comes from, but if it's an internal definition, it might be worth just flattening this into that  variable.

Reply:
+1 for a named variable. Does the `multiple` value only reflect correctly for the Block-Editor context? Like image block vs gallery block?

Maybe:
```
// Classic behavior allows for insertion of one or many media images depending. Single selection is an image, and multiple creates a gallery
const isSingle = ! multiple || isAddingMediaWithClassicBlock;
```

Reply:
Correct - so the `multiple` variable is set to false in the classic editor context. I'll make an update to make this variable more clear :)

Reply:
Instead of making another variable name, I figured it was even more clear to just set the `multiple` state value to true at the same point when it would for a normal block.